Logstash/logstash-input-azuretopicthreadable/lib/logstash/inputs/azuretopicthreadable.rb

```diff
@@ -47,9 +58,12 @@ def process(output_queue, pid)
           codec.decode(message.body) do |event|
           decorate(event)
           output_queue << event
-        end # codec.decode
-        # delete the message after reading it
-        azure_service_bus.delete_subscription_message(message)
+          # delete the message after reading it
+          azure_service_bus.delete_subscription_message(message)
+        end
```

Comment:
I tired to use `else` here but for some reason it throws a syntax error. I'm not a ruby guru and  haven't been able to figure out what is wrong.  Suggestions welcomed. 

Here's what I have,
```
        if message
          # decoding returns a yield
          codec.decode(message.body) do |event|
          decorate(event)
          output_queue << event
          # delete the message after reading it
          azure_service_bus.delete_subscription_message(message)
        else
          Stud.stoppable_sleep(@thread_sleep_time) { stop? } #topic is probably empty. sleep. 
        end
```